gprofiler/profilers/node.py

```diff
@@ -186,6 +186,7 @@ def create_debugger_socket(nspid: int, ns_link_name: str) -> WebSocket:
     sock.settimeout(10)
     _validate_ns_node(sock, ns_link_name)
     _validate_pid(nspid, sock)
+    logger.info(f"Created debugger socket for pid: {nspid} (nodejs)")
```

Comment:
Prefer to pass arguments as extras instead of embedding them in the message itself. For example, as done here https://github.com/Granulate/gprofiler/blob/c95af658df7723ce0a9e74e67c94e1e6224c16ac/gprofiler/profilers/java.py#L247